src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -103,6 +103,7 @@ static PcsStartup()
                             (sub.SourceRepository == entity.Build.GitHubRepository || sub.SourceDirectory == entity.Build.AzureDevOpsRepository) &&
                             JsonExtensions.JsonValue(sub.PolicyString, "lax $.UpdateFrequency") == ((int)UpdateFrequency.EveryBuild).ToString())
                         // TODO (https://github.com/dotnet/arcade-services/issues/3880)
+                        .ToList()
```

Comment:
Is this `ToList()` needed here?

Reply:
Yes, this first one loads it from the DB and only then we filter it because that method call there cannot be done in the DB